src/models/predict_model.py

```diff
@@ -0,0 +1,55 @@
+# models/predict_model.py
+
+
+# UNSURE ABOUT THIS SECTION :: DO I NOT WANT TO JUST HAVE THIS OUTPUT WHATEVER
+# INPUT ONE GIVES INSTEAD OF DOING IT FOR THE TEST DATA ONLY ?
```

Comment:
It really depends on what the purpose of this file is supposed to be. If it is just to give us statistics for the model, this is okay. If the purpose is to predict on new data, then we indeed should predict on given input in my opinion